app/queues/reconciler/reconciler-queue.server.ts

```diff
@@ -28,7 +27,9 @@ export const addReconcilerJob = async () => {
     // This is important because multiple repeatable jobs can exist and they persist
     // within redis (even with the same key)
     const repeatableJobs = await reconcilerQueue.getRepeatableJobs();
-    Promise.all(repeatableJobs.map(({ key }) => reconcilerQueue.removeRepeatableByKey(key)));
+    repeatableJobs.forEach(async ({ key }) => {
```

Comment:
I'd still use `await Promise.all()` and `.map` here, but if you want a loop, use `for-of` you're going to `await` inside.

Reply:
You are right, Updating back, adding the missing await